app/README.md

```diff
@@ -0,0 +1 @@
+# Converter app
```

Comment:
There are two readme files. You can update this one instead and delete the duplicate files in the Index folder. Then just move the contents in the app folder to the main folder.

Reply:
Yeah, that could work.